analyses/tcd/metrics/TCD_trigger_report.Rmd

```diff
@@ -0,0 +1,432 @@
+
+# trigger A info  
+name_a <- "Trigger1" # what name is the trigger listed as in the perf_metrics_table.csv files?
+type_a <- 'Predictive' # Predictive or Observational?
+monitored_area_a <- "Central Chad"
+timepoints_a<- 'Mar, Apr' # eg: 'Jan, Feb, Mar'
+package_a <- 'Package 1' # eg: 'Package 1'
+funding_a <- 3.71 # input number only, no $ sign or 'm' for million. Formatting will be done automatically.
+targeted_window_a <- 'Jul-Sep' # 3-letter names for first and last month. eg: 'Jul-Sep'. Forecasted period or period over which obs data are computed.
+lead_time_a <- '4-3 months' # 'eg: 7-5 months', Change units as needed. Include all lead times if multiple activation timepoints
+data_source_a <- 'IRI seasonal forecast' # name of product + provider
+trigger_a <- '42.5% below average probability'
+hist_period_a <- "2018 - present" # period for which historical data is available
+monitored_by_a <- 'CHD' # owner of monitoring responsibilities
+
+# trigger B info  
+name_b <- "Trigger2"
+type_b <- 'Predictive'
+monitored_area_b <- "Central Chad"
+timepoints_b <- 'May, Jun'
+package_b <- 'Package 2'
+funding_b <- 1.66
+targeted_window_b <- 'Jul-Sep'
+lead_time_b <- '2-1 months'
+data_source_b <- 'IRI seasonal forecast'
+trigger_b <- '42.5% below average probability'
+hist_period_b <- "2018 - Present"
+monitored_by_b <- 'CHD'
+  
+# trigger C info  
+name_c <- "Trigger3"
+type_c <- 'Observational'
+monitored_area_c <- "Central Chad"
+timepoints_c <- 'Sep'
+package_c <- 'Package 2'
+funding_c <- 4.63
+targeted_window_c <- 'Apr-Sep'
+lead_time_c <- 'N/A'
+data_source_c <- 'Biomass'
+trigger_c <- 'Seasonal anomaly below 80%'
+hist_period_c <- "2000 - Present"
+monitored_by_c <- 'CHD'
+
+# Compile list of triggers to be included in the report
+trigger_list <- c(name_c) # limited due to issues with historical data in Triggers 1 and 2
+scenario_list <- c() # No scenarios given limits of data
+
+# Table footnotes
+trigger_table_footnote <- "Expected annual cost computed using most likely probability of activation (central value). All triggers can reach their threshold and activate independently from one another. Triggers 1 and 2 did not have sufficient data for testing performance on Chad, but likelihood of activation tested on global data. Trigger 3 performance assessed by comparing recommended activations with historical bad years (i.e., years with a shock). 'Bad years' were identified through convergence of evidence reviewed from historical CERF allocations, FEWS NET reports, EM-DAT data, World Bank statistics and the 2015 and 2017 multi-risk contingency plans for Chad."
+
+full_atv_definition <- "A full activation occurs when all three triggers met and all packages are disbursed."
+package1_atv_definition <- "Package 1 is activated when Trigger 1 is met."
+package2_atv_definition <- "Package 2 is activated when Trigger 2 is met."
+package3_atv_definition <- "Package 3 is activated when Trigger 3 is met."
```

Comment:
The table lists Trigger 3 with Package 2. Is there a Package 3?

Reply:
Yep, good catch! We have three separate packages.